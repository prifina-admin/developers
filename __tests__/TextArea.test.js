import React, {createRef} from "react";
import renderer from "react-test-renderer";
import ThemeProvider from "../src/theme/ThemeProvider";
import TextArea from "../src/TextArea";

test("TextArea renders correctly", () => {
    const tree = renderer.create(
        <ThemeProvider>
            <TextArea minHeight={"100px"} />
        </ThemeProvider>).toJSON();
    expect(tree).toMatchSnapshot();
});

test("Expanded TextArea renders correctly", () => {
    const tree = renderer.create(
        <ThemeProvider>
            <TextArea expand />
        </ThemeProvider>).toJSON();
    expect(tree).toMatchSnapshot();
});
