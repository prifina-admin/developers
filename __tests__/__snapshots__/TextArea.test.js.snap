// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expanded TextArea renders correctly 1`] = `
<div
  className="ThemeProvider__Base-sc-1rk52n4-0 emGfPA"
>
  <textarea
    className="Input__InputElement-sc-1ezb9jk-0 epsbol TextArea__StyledExpandTextArea-sc-1wkrv2f-1 iUVOKg"
    height="1.25rem"
    onInput={[Function]}
    rows="1"
    type="text"
    width="100%"
  />
</div>
`;

exports[`TextArea renders correctly 1`] = `
<div
  className="ThemeProvider__Base-sc-1rk52n4-0 emGfPA"
>
  <textarea
    className="Input__InputElement-sc-1ezb9jk-0 iQtNIx TextArea__StyledTextArea-sc-1wkrv2f-0 hykFNy"
    rows="1"
    type="text"
    width="100%"
  />
</div>
`;
